slices/base-files.yaml

```diff
@@ -25,6 +25,8 @@ slices:
     contents:
       /lib:
       /usr/lib/:
+      /lib64: {arch: [amd64,ppc64el]}
```

Comment:
Open question, would it be better to specify the symlink in Chisel instead of relying in the one from the tarball? I find the difference between `lib64/` (extract folder) and `lib64` (extract symlink) not that clear.

Reply:
It is definitely something we can discuss. I'm not sure we should let chisel make that decision though, cause then you'd lose the freedom of choice on whether to include the symlinks or not. We also have the `symlink` path property, which probably could be used here to reduce the confusion, if needed